development/tools/jobs/tester/tester.go

```diff
@@ -83,7 +83,7 @@ type jobRunner interface {
 
 // GetAllKymaReleaseBranches returns all supported kyma release branches
 func GetAllKymaReleaseBranches() []string {
-	return []string{"release-0.6"}
+	return []string{"release-0.7"}
```

Comment:
IMO we should have 2 release branches supported

Reply:
done